lib/rubocop/cop/rails/enum_syntax.rb

```diff
@@ -24,7 +24,10 @@ class EnumSyntax < Base
         MSG = 'Enum defined with keyword arguments in `%<enum>s` enum declaration. Use positional arguments instead.'
         MSG_OPTIONS = 'Enum defined with deprecated options in `%<enum>s` enum declaration. Remove the `_` prefix.'
         RESTRICT_ON_SEND = %i[enum].freeze
-        OPTION_NAMES = %w[prefix suffix scopes default].freeze
+
+        # From https://github.com/rails/rails/blob/5c0b7496ab32c25c80f6d1bdc8b32ec6f75ce1e4/activerecord/lib/active_record/enum.rb#L222
```

Comment:
I like this comment.
Can you tweak to the following? This clarifies the reference version.

```suggestion
        # From https://github.com/rails/rails/blob/v7.2.1/activerecord/lib/active_record/enum.rb#L231
```

Reply:
I'll do tag references for these in the future, that's much better :+1: